oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -163,6 +175,10 @@ private boolean listElementStartsWith(List<String> aList, String charSeq) {
                 continue;
             }
 
+            if (localOnly && (type.startsWith("nfs") || type.equals("cifs"))) {
```

Comment:
Probably should add `smbfs` and `sshfs` for completeness (here and other places these are used).  Actually found this list...

`afs, cifs, smbfs, sshfs, ncpfs, ncp, nfs, nfs4, gfs, gds2, glusterfs`

Plus `afp` on mac, although looks like we handle that separately.

Might want a protected static `isNetworkFsType(String type)` method in the abstract parent.